Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js

```diff
@@ -98,7 +98,35 @@ var sendRequest = function(method, uri, body, raw) {
     }
 };
 
-var deleteIncidents = function(ids_to_delete) {
+function reduce_one_entry(data, keep_fields) {
+    var new_d = {};
+    for (var field_index = 0; (field_index < keep_fields.length); field_index += 1) {
+        var field = keep_fields[field_index];
+        if (data[field]) {
+            new_d[field] = data[field];
+        }
+    }
+    return new_d;
+}
+
+function reduce_data(data, fields_to_keep) {
+    if ((data instanceof Array)) { // is array
```

Comment:
no need for the 2nd parenthesis or the comment.

Reply:
Done